vignettes/implementing-cookies.Rmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "Implementing cookie consent"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Implementing cookie consent}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Cookie consent is a mandatory requirement for any dashboard when using cookies 
+and typically all DfE dashboards will be using cookies as part of tracking with
+Google Analytics and so will require cookie consent.
+
+> Note that to activate Google Analytics on your data dashboard, you will need 
+to contact the Statistics Development team at 
+[statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk) and request a Google Analytics key.
+
+dfeshiny provides a standard GDS compliant banner, a cookie information panel 
+and the underlying code required to allow end users to give or withhold their 
+consent for cookies to be used. If they withhold their consent, then tracking 
+with Google Analytics will be halted.
```

Comment:
What are the implications of this for analysts? Do they need to know?

Reply:
Added info in brackets.